### PR TITLE
Adds basic trailer support to vehicles

### PR DESCRIPTION
**Pull request by Iamgoofball**
i006593

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Great PR desc. Very informative.

**Comment by epochayur** [comment]
>thumbing up your own PR

thumb revoked

**Comment by Anonmare** [comment]
>People who thumbs up their own posts
🤢

**Comment by PKPenguin321** [comment]
>github reactions

**Comment by ShiggyDiggyDo** [comment]
if you don't hooray this post your mother will die in her sleep tonight

**Comment by Bawhoppen** [comment]
That's just not right @ShiggyDiggyDo

**Comment by Buggy123** [comment]
>People who thumbs up their own posts... are great!

**Comment by ExcessiveUseOfCobblestone** [comment]
Is that the truck in Big Trouble in Little China?

File: code/modules/vehicles/_vehicle.dm [hunk 1]
```diff
@@ -111,7 +112,14 @@
 	if(lastmove + movedelay > world.time)
 		return FALSE
 	lastmove = world.time
-	return step(src, direction)
+	if(trailer)
+		var/dir_to_move = get_dir(trailer.loc, loc)
+		var/did_move = step(src, direction)
+		if(did_move)
+			step(trailer, dir_to_move)
```
Comment (optimumtact) [inline code/modules/vehicles/_vehicle.dm]: you should call move on the trailer so it gets proper movement

Comment (RemieRichards) [reply code/modules/vehicles/_vehicle.dm]: (step calls move)

**Pull request merged**